certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -3,676 +3,2194 @@
+        "consumes": [
+          "text/plain",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "XML",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "XML",
+            "description": "XML"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
+        "summary": "Convert XML to JSON (deprecated)",
+        "description": "Convert XML to JSON"
+      }
+    },
+    "/ConvertXmlToJsonV2": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "ConvertXmlToJsonV2",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "xmlData",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/XmlData"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "XML",
+            "description": "XML"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/JsonResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "important",
+        "summary": "Convert XML to JSON",
+        "description": "Convert XML to JSON"
+      }
+    },
+    "/ConvertJsonToXml": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "ConvertJsonToXml",
+        "consumes": [
+          "text/plain",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "JSON",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "JSON",
+            "description": "JSON"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
```

Comment:
@stsiek The operation has been hidden. This change breaks the designer for existing flows.

Reply:
we would like to set this action to deprecated, so it is set to "internal". we had understood that it can still be used for existing flows but is no longer visible when adding new actions. We also tested it and it works fine.